contrib/buildrefactor/tests/python/pants_test/contrib/buildrefactor/test_meta_rename_integration.py

```diff
@@ -0,0 +1,30 @@
+# coding=utf-8
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+from pants_test.pants_run_integration_test import PantsRunIntegrationTest
+
+
+class MetaRenameIntegrationTest(PantsRunIntegrationTest):
+  def test_meta_rename(self):
+
+    pre_dependees = self.run_pants(['dependees',
```

Comment:
1. assert run passed `pants_test.pants_run_integration_test.PantsRunIntegrationTest#assert_success`
2. nit: rename `pre_dependees` to `pre_dependees_run`

ditto for post.